hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -5028,73 +5028,70 @@ public void testCheckAndDeleteWithCompareOp() throws IOException, InterruptedExc
         FAMILY);
     TEST_UTIL.waitTableAvailable(TableName.valueOf("testCheckAndDeleteWithCompareOp"), 10000);
 
-    Put put2 = new Put(ROW);
-    put2.add(FAMILY, QUALIFIER, value2);
-    table.put(put2);
+    Put bbbb = new Put(ROW);
+    bbbb.add(FAMILY, QUALIFIER, value2);
 
-    Put put3 = new Put(ROW);
-    put3.add(FAMILY, QUALIFIER, value3);
+    Put cccc = new Put(ROW);
+    cccc.add(FAMILY, QUALIFIER, value3);
 
     Delete delete = new Delete(ROW);
     delete.deleteColumns(FAMILY, QUALIFIER);
 
     // cell = "bbbb", using "aaaa" to compare only LESS/LESS_OR_EQUAL/NOT_EQUAL
     // turns out "match"
+    table.put(bbbb);
```

Comment:
Curious to know if keeping this `put` operation on this line is a code improvement or has something to do with flaky behaviour?

Reply:
Just for readability only, this was originally in line 5033. I just separated it from initialization.